source/tool/chipsec/helper/linux/helper.py

```diff
@@ -78,6 +80,8 @@
 class LinuxHelper(Helper):
 
     DEVICE_NAME = "/dev/chipsec"
+    MODULE_NAME = "chipsec"
+    GET_PAGE_IS_RAM = True
```

Comment:
Is this attribute necessary? 
We can always try to find the symbol. If we find it, forward it to the driver. Otherwise, let the driver find it. If the driver fail, report the error and die.
If we want to give the user the choice to decide on that option, we should surface this to chipsec_utils and chipsec_main.